### ISSUE TITLE
Unable to edit any image caption: IndexError: list index out of range

### ISSUE THREAD

**Issue by flotos** [toshiaki1729/stable-diffusion-webui-dataset-tag-editor#16]
Hi, when I try to edit an image caption "apply changes to selected image", then switch image, then go back to the first image the caption I had saved is gone. Pressing "save all changes" does not save the caption to .txt file.

Is get this error output :

```
Loading Completed: 66 images found
Traceback (most recent call last):
  File "C:\Users\MYNAME\Documents\stablediffusion\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 284, in run_predict
    output = await app.blocks.process_api(
  File "C:\Users\MYNAME\Documents\stablediffusion\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 980, in process_api
    inputs = self.preprocess_data(fn_index, inputs, state)
  File "C:\Users\MYNAME\Documents\stablediffusion\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 891, in preprocess_data
    processed_input.append(block.preprocess(inputs[i]))
IndexError: list index out of range
```

I have 66 images in my dataset.

**Comment by toshiaki1729** [comment]
Thank you so much!
I finally managed to find what is happening and fixed it.

**Comment by toshiaki1729** [comment]
Thank you so much for your report!
This issue is fixed in d46f1a5520b9ca4d105144e98007dd69ee584013.
